config/katello-installer.yaml

```diff
@@ -45,6 +45,9 @@
     :foreman::plugin::templates:
       :dir_name: foreman
       :manifest_name: plugin/templates
+    :foreman::plugin::remote_execution:
+      :dir_name: foreman
+      :manifest_name: plugin/remote_execution
```

Comment:
Not all plugins have params. This allows kafo to load the defaults from the params file if it's used.